cloudeon-server/src/main/java/org/dromara/cloudeon/utils/ScriptRunner.java

```diff
@@ -1,19 +1,19 @@
 /*
- * Copyright 2004-2020 the original author or authors.
```

Comment:
As it is mentioned "Copy from Apache Dolphinscheduler", why change the header? it's illegal!